terraform/modules/iam/main.tf

```diff
@@ -0,0 +1,66 @@
+##########################################################
+# IAM
+#
+# Spree application user
+##########################################################
+resource "aws_iam_user" "spree" {
+  name = "spree-app-user"
+}
+
+resource "aws_iam_access_key" "spree" {
+  user = aws_iam_user.spree.name
+}
+
+# Create System Parameters as we need these to inject as secrets into ECS
+resource "aws_ssm_parameter" "aws_access_key_id" {
+  name      = "/bat/${lower(var.environment)}-aws-access-key-id"
+  type      = "SecureString"
+  value     = aws_iam_access_key.spree.id
+  overwrite = true
+}
+
+resource "aws_ssm_parameter" "aws_secret_access_key" {
+  name      = "/bat/${lower(var.environment)}-aws-secret-access-key"
+  type      = "SecureString"
+  value     = aws_iam_access_key.spree.secret
+  overwrite = true
+}
+
+resource "aws_iam_user_policy" "spree" {
+  name = "spree-user-policy"
+  user = aws_iam_user.spree.name
+
+  # Terraform's "jsonencode" function converts a
+  # Terraform expression result to valid JSON syntax.
+  policy = jsonencode({
```

Comment:
I like this new way of defining the policy statements, much clearer than the old heredoc style 👍